source/beamline/BioXAS/BioXASSSRLMonochromator.h

```diff
@@ -62,9 +62,11 @@ class BioXASSSRLMonochromator : public BioXASMonochromator
 	/// Returns the brake status control.
 	AMControl* brakeStatusControl() const { return brakeStatus_; }
 	/// Returns the bragg control.
-	AMControl* braggControl() const { return bragg_; }
+	AMControl* braggControl() const { return stepBragg_; }
```

Comment:
I'm a little confused about this stepBragg and encoderBragg.  Are you removing it or is this for some other purpose?  I'm also a little confused with some of the switches throughout a couple of these classes.  Some get the step bragg and others get the encoder bragg.  And in this particular  location, there is a `braggControl()`, `stepBragControl()`, and `encoderBraggControl()`, is there a reason for keeping all three?